### PR TITLE
INTYGFV-16587: Implemented GetQuestionsAggregator.

### PR DESCRIPTION
**Pull request by ChrilleSigerhed**

### CODE REVIEW

**Review by mww-sopra-steria — APPROVED** [review]
Looking good! Few minor comments to consider before merge, but it's approved! Reach out if any need to discuss.

File: web/src/main/java/se/inera/intyg/webcert/web/csintegration/integration/dto/GetCertificteFromMessageResponseDTO.java
Comment (ChrilleSigerhed) [reply web/src/main/java/se/inera/intyg/webcert/web/csintegration/integration/dto/GetCertificteFromMessageResponseDTO.java]: Ops, good catch!

**Pull request merged**
